pages/api/verify/[uniqueString].ts

```diff
@@ -0,0 +1,33 @@
+import nc from "next-connect";
+
+import userService from "../../../src/server/services/user.service";
+
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const handler = nc<NextApiRequest, NextApiResponse>({}).get(
+  async (req, res) => {
+    try {
+      const { uniqueString } = req.query;
+
+      const user = await userService.findByUniqueString(
```

Comment:
The call of `findByUniqueString` and `addConfirmation` methods should be combined in a single service call.